docs/modules/operation/pages/provisioning/detectors/FtpDetector.adoc

```diff
@@ -0,0 +1,31 @@
+= FTP Service Detector
+
+The FtpDetector is able to validate ftp connection dial-up processes.
+The detector can test ftp server on multiple ports and specific login data.
+
+== Detector facts
+
+[options="autowidth"]
+|===
+| Implementation | `org.opennms.netmgt.provision.detector.simple.FtpDetector`
+|===
+
+== Configuration and Use
+
+.Parameters for the FTP Service Detector
+[options="header, %autowidth"]
+|===
+| Parameter            | Description   | Required | Default value
+| `port`               | Port to query | optional | `21`
+| `multilineIndicator` | ???           | optional | `-`
```

Comment:
???

Reply:
Feel free to add a useful description :-) I am not sure what exactly this is doing. The [monitor](https://github.com/OpenNMS/opennms/blob/develop/docs/modules/operation/pages/service-assurance/monitors/FtpMonitor.adoc) does not have this parameter.

Reply:
```suggestion
```